<?xml version="1.0"?>
<launch>
  <!-- Should be false -->
  <arg name="vis" default="False" />
  <arg name="pf" default="True" />
  <!--
  <arg name="pf_camera_topic" default="/goal_detection/goal_detect_msg" />
  -->
  <arg name="pf_camera_topic" default="/tf_object_detection/object_detection_world_filtered" />
  <group ns="pf_localization">
	   <rosparam command="load" file="$(find pf_localization)/config/2023_params.yaml" />
	   <node name="pf_localization" pkg="pf_localization" type="pf_localization_node" output= "screen" if="$(arg pf)">
      <remap from="/goal_detection/goal_detect_msg" to="$(arg pf_camera_topic)" />
    </node>
    <node name="pf_visualization" pkg="pf_localization" type="pf_vis.py" output="screen" if="$(arg vis)">
    </node>
  </group>


</launch>
